examples/qml_features/rust/src/signals.rs

```diff
@@ -49,6 +51,28 @@ pub mod ffi {
             // Emit a signal to QML stating that we have disconnected
             self.as_mut().emit(Connection::Disconnected);
         }
+
+        #[qinvokable]
+        pub fn toggle_logging(mut self: Pin<&mut Self>) {
+            if let Some(connections) = self.as_mut().connections_mut().take() {
+                for conn in connections {
+                    conn.disconnect();
+                }
+            } else {
+                let connections = [
+                    self.as_mut().on_connected(|url| {
```

Comment:
should we have the `Qt::ConnectionType` as an argument here ? Eg so you can do `Qt::DirectConnection` or `Qt::QueuedConnection` rather than always assuming `Auto` ?

Reply:
yes

Reply:
OK wonder if it should be the first or last argument, in Qt it's the last, but then it's after the closure, maybe that's fine though

Reply:
Done, added them as the last parameter.